packages/cyberstorm/src/components/DropDownItem.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/DropDownItem.module.css";
+
+export interface DropDownItemProps {
+  label?: string;
+  leftIcon?: ReactNode;
+  rightIcon?: ReactNode;
+  colorScheme?: "default" | "defaultDark" | "primary";
```

Comment:
Since `DropDown` attempts to pass its own `colorScheme` to the options, it could be a good idea to define the color schemes in one of the components and import into the other. Since my other suggestions already require importing stuff from here to `DropDown`, it would make sense to define the schemes here (assuming my suggestions are actually implemented).

Reply:
I'd let the DropDownitem handle it's props separately, to not depend too much on each other.
Default colorScheme is used if the passed one isn't valid.
I think in the future some list of colorSchemes might be implemented for all of the components and the shcemes would basicly be in line with each other. Feel free to change my mind.